paper/user_study.tex

```diff
@@ -49,8 +49,128 @@ \subsection{Results}
+        & 4. When dealing with an array, the naming format for object name should be improved, replacing the default numerical labels (0, 1, 2, etc.) with a more descriptive way of showing the object. & 4. Improvement can be achieved by replacing simple numerical labels (0, 1, 2, etc.) for objects with a naming convention like "PropertyName[0]" and so forth. \\
+        \cline{2-3}
+        & 5. Additionally, when clicking a new property, it should not automatically create a new field, especially when there is already one field by default for adding the first property. & 5. Will be Consider for future work. \\
+        \cline{2-3}
+        & 6. In the schema editor, it would be more consistent and user-friendly if the functionality to give a name to a property were located on the right side of the GUI editor panel, eliminating the need to click on the property first in order to rename it. & 6. We have retained the property naming on the right end but improved it with an intuitive design. When clicking on a new property in the schema editor, it is now highlighted to prompt the user to change the name. \\
+        \cline{2-3}
+        & 7. The search function for locating specific properties lacks clarity at first glance. It should provide an immediate response and extend to nested levels, rather than merely highlighting the higher-level property in yellow. & 7. The search functionality has been enhanced. It now provides a list of properties containing the keyword, and upon clicking, it highlights the selected property while opening the nested levels in the GUI editor panel for improved usability. \\
+        \hline
+    \end{tabular}
+    \caption*{User Study Feedback and Resolution (Continued)}
+\end{table*}
+
+\begin{table*}
+    \centering
+    \small % Reduce font size
+    \setlength{\extrarowheight}{5pt} % Add extra vertical space
+    \renewcommand{\arraystretch}{1.5} % Adjust the vertical spacing between rows
+    \begin{tabular}{|c|p{0.4\linewidth}|p{0.4\linewidth}|}
+        \hline
+        User Study 3 & 1.Still I have to right & 1. bla bla \\
+        \cline{2-3}
+
+        \hline
+    \end{tabular}
+    \caption*{User Study Feedback and Resolution (Continued)}
+\end{table*}
+
+\begin{table*}
+
+    \centering
+    \small % Reduce font size
+    \setlength{\extrarowheight}{5pt} % Add extra vertical space
+    \renewcommand{\arraystretch}{1.5} % Adjust the vertical spacing between rows
+    \begin{tabular}{|c|p{0.4\linewidth}|p{0.4\linewidth}|}
+        \hline
+        User Study 4 & 1.Modifying or renaming a 'new property' in Schema Editor does not appear to take effect when double-clicking on it. & 1. Renaming of the new property is made possible by double-clicking on 'newProperty' itself. \\
+        \cline{2-3}
+        & 2. Upon selecting a property type in the schema editor, the type of the child property should be automatically chnaged. & 2. The type of the child element is automatically adjusted according to parent type. \\
```

Comment:
```suggestion
        & 2. Upon selecting a property type (e.g., "String property") in the schema editor, the type of the child property should be automatically changed. & 2.  we completely overhauled our own meta-schema, so that this specific scenario cannot occur anymore. \\
```